src/wp-includes/widgets/class-wp-widget-media-gallery.php

```diff
@@ -154,7 +154,7 @@ public function form( $instance ) {
 		$instance = array_merge( wp_list_pluck( $this->get_instance_schema(), 'default' ), $instance );
 
 		$title             = ! empty( $instance['title'] ) ? $instance['title'] : '';
-		$ids               = ! empty( $instance['ids'] ) ? $instance['ids'] : '';
+		$ids               = ! empty( $instance['ids'] ) ? implode( ',', $instance['ids'] ) : '';
```

Comment:
Works well, only suggestion is that the default should perhaps be `array()` rather than `''`.

Reply:
Yes, I tried that too! 😉

The default needs to be a string, not an array (hence the `implode()`), so it should be `''`.

Reply:
Of course! You are correct, too long staring at this code today is my only excuse!